XamarinCommunityToolkitSample.Android/Xamarin.CommunityToolkit.Sample.Android.csproj

```diff
@@ -53,10 +54,10 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Xamarin.Forms" Version="4.8.0.1269" />
+    <PackageReference Include="Xamarin.Forms" Version="4.8.0.1451" />
```

Comment:
is there any reason for reducing forms version?

Reply:
I updated PancakeView (I added support for GTK and other minor improvements: https://github.com/sthewissen/Xamarin.Forms.PancakeView/pull/145). and it depends on version 1451